mypy/semanal.py

```diff
@@ -47,7 +47,6 @@
   daemon strips and reuses existing ASTs (to improve performance and/or
   reduce memory use).
 """
-
```

Comment:
was this intentional

Reply:
I don't even understand this diff, is it a deleted line?

Reply:
oh ok, it was changed twice, I've fixed it.